lib/pliny/metrics/backends/librato.rb

```diff
@@ -0,0 +1,50 @@
+begin
+  require "librato/metrics"
+  require "concurrent"
+rescue LoadError => error
+  puts <<-MSG
+! #{error.message}
+! Please add:
+!   gem "librato-metrics"
+!   gem "concurrent-ruby"
+! to your gemfile and run `bundle install` if you wish to use the Librato
+! metrics backend
+
+  MSG
+
+  raise
+end
+
+module Pliny
+  module Metrics
+    module Backends
+      class Librato
```

Comment:
I don't love that this is a class and the Logger backend is a module with class methods. Unfortunately we need an instance of this so that we can attach all of the `Concurrent::Async` stuff. Open to suggestions on how to make this consistent (maybe just require that `Backends::Logger` is a class too?).